src/plugins/usage_collection/README.md

```diff
@@ -110,22 +119,52 @@ export function registerMyPluginUsageCollector(
   }
 
   // create usage collector
-  const myCollector = usageCollection.makeUsageCollector({
-    type: MY_USAGE_TYPE,
-    isReady: () => typeof getSavedObjectsRepository() !== 'undefined',
-    fetch: async () => {
-      const savedObjectsRepository = getSavedObjectsRepository()!;
-      // get something from the savedObjects
-
-      return { my_objects };
-    },
-  });
+  const myCollector = usageCollection.makeUsageCollector<Usage>(...)
 
   // register usage collector
   usageCollection.registerCollector(myCollector);
 }
 ```
 
+## Schema Field
+
+The `schema` field is a proscribed data model that eliminates maintenance of mappings in the telemetry cluster. Simply define a schema field that specifies every possible field reported by the collector. Whenever the `schema` field is set or changed. Please run `node scripts/telemetry_check.js --fix` to update the stored schema json files.
```

Comment:
```suggestion
The `schema` field is a proscribed data model that assists with detecting changes in usage collector payloads. Simply define a schema field that specifies every possible field reported by the collector. Whenever the `schema` field is set or changed. Please run `node scripts/telemetry_check.js --fix` to update the stored schema json files.
```

Reply:
re my suggestion: we should rather not couple the usage data schema with the mappings in the telemetry cluster because these are likely not going to be 1 to 1.